repair/repair.cc

```diff
@@ -1100,7 +1107,8 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
 
     // Do it in the background.
     (void)repair_tracker().run(id, [this, &db, id, keyspace = std::move(keyspace),
-            cfs = std::move(cfs), ranges = std::move(ranges), options = std::move(options), ignore_nodes = std::move(ignore_nodes)] () mutable {
+            cfs = std::move(cfs), ranges = std::move(ranges), options = std::move(options), ignore_nodes = std::move(ignore_nodes),
+            task] () mutable {
```

Comment:
no need to break line.
And if you do, please start the new one two tab-indents deeper as per our coding style